gcp/rakefiles/test.rake

```diff
@@ -0,0 +1,23 @@
+desc '[TEST] Run Locust swarm against Preferences service in current cluster'
+task :test_preferences => [:set_vars] do
+  sh "#{@exekube_cmd} rake xk[' \
+    xk down live/dev/locust && \
+    sleep 30 && \
+    TF_VAR_locust_target_host=http://preferences.$TF_VAR_domain_name \
+    TF_VAR_locust_script=preferences.py \
+    xk up live/dev/locust',skip_infra,skip_secret_mgmt]"
+end
+
+desc '[TEST] Run Locust swarm against Flowmanager service in current cluster'
+task :test_flowmanager => [:set_vars] do
+  sh "#{@exekube_cmd} rake xk[' \
+    xk down live/dev/locust && \
+    sleep 30 && \
```

Comment:
Can you add `echo Sleeping 30s...` here so the user doesn't think the process is stuck?

Reply:
See next tread.